tests/test_file_logging.cpp

```diff
@@ -34,8 +34,8 @@ TEST_CASE("flush_on", "[flush_on]]")
     logger->info("Test message {}", 1);
     logger->info("Test message {}", 2);
 
-    REQUIRE(file_contents(filename) == std::string("Should not be flushed\nTest message 1\nTest message 2\n"));
-    REQUIRE(count_lines(filename) == 3);
+    require_message_count(filename, 3);
+    REQUIRE(file_contents(filename) == std::string("Should not be flushed") + spdlog::details::os::default_eol + std::string("Test message 1") + spdlog::details::os::default_eol + std::string("Test message 2") + spdlog::details::os::default_eol);
```

Comment:
please use fmt::format for a cleaner version:

```c++
using spdlog::details::os::default_eol;
REQUIRE(file_contents(filename) == fmt::format("Should not be flushed{}Test message 1{}Test Message 2{}",
    default_eol, default_eol, default_eol);
```

Reply:
I see that this is a cleaner notation. But the style I'm currently using is also used in several other places. I won't gt time to make this improvement, is it ok to leave this out?

Reply:
There is no rush, please take your time. I will wait with the merge.